css/style.css

```diff
@@ -2587,6 +2594,39 @@ a.next.page-numbers:before {
   font-weight: 600;
 }
 
+.dark-mode .contact h4.wp-block-heading, 
+.dark-mode .contact ul.no-bullets li a:hover {
+  color: var(--theme-secondary) !important;
+}
+
+/* About page */
+
+.about-list .wp-block-column:first-child {
+  padding-left: var(--spacing-14);
```

Comment:
Do we have any element with class 'about-list' right now?

Reply:
Yes, Martin (wp-block-group about-list pv--40 is-layout-constrained wp-block-group-is-layout-constrained)

Reply:
Noelia, I don't get it as there is already page I created: https://news.mongabaydev.wpengine.com/about and it's linked in relevant places as well as placed in testing column in Asana. There is no need duplicating it which is why extra classes confused me.